namespaces/live.cloud-platform.service.justice.gov.uk/hmpps-official-visits-dev/resources/variables.tf

```diff
@@ -32,6 +32,12 @@ variable "team_name" {
   default     = "hmpps-move-and-improve"
 }
 
+
+variable "service_area" {
+  description = "Service area responsible for this service"
+  default     = "Official Visits"
+}
```

Comment:
I think this needs to be an exact match ie.. `Activities and visits` - because it matches a real service area description in other resources. Whatever it was set to on the example PR given please.